app/views/kaminari/_page.html.haml

```diff
@@ -0,0 +1,10 @@
+-#  Link showing page number
```

Comment:
Comment should have a space after the `#`<br>8 consecutive comments can be merged into one